Documentation/Blazorise.Docs/Components/Toc.razor

```diff
@@ -0,0 +1,8 @@
+﻿@inherits BaseAfterRenderComponent
```

Comment:
Is this blazorise component worth it?

Reply:
Leftover from refactoring. Removed it now.